ci/2.7.pylintrc

```diff
@@ -7,19 +7,12 @@
 # bad-open-mode<W1501>: (hi-pri) Python supports: r, w, a[, x] modes with b, +, and U (only with r) options.
 # bad-super-call<E1003>: (hi-pri) Used when another argument than the current class is given as first argument of the super builtin.
 # bare-except<W0702>: (hi-pri) Used when an except clause doesn't specify exceptions type to catch.
-# blacklisted-name<C0102>: (needs review) Used when the name is listed in the black list (unauthorized names).
 # cell-var-from-loop<W0640>: (hi-pri) A variable used in a closure is defined in a loop. This will result in all closures using the same value for the closed-over variable.
-# cyclic-import<R0401>: (hi-pri) Used when a cyclic import between two or more modules is detected.
```

Comment:
We should not remove this error, this doesn't look cosmetic

Reply:
There were no instances of this suppression, this comment was out of date.

Reply:
>There were no instances of this suppression

Wont it still be good to have this check incase we unknowingky add a cyclic dependency in the future?

Reply:
Keep this comment if we're not ignoring this error

Reply:
The original intention of those comments was to document the text description of the message, but they are already out of sync with the code and are likely to get more out of sync and we make additional changes. I was inclined to remove them altogether, but the rcfile for 2.7 has some infor wrt python versions that may be useful.

Reply:
BTW, the description can be obtained from the command line using "pylint --list-msgs"

Reply:
Hmm interesting, I thought the idea behind adding the descriptions here was to have a one stop shop for all the messages we were ignoring in our code. 

Out of curiosity, what exactly do you mean by out of sync? Do we have some pylint warnings which are there in the code but not in our .rc file and vice versa?

Reply:
>I thought the idea behind adding the descriptions here was to have a one stop shop for all the messages we were ignoring in our code.
Exactly, but they are already out of sync. The one stop shop can instead be  "pylint --list-msgs"

> what exactly do you mean by out of sync? 
Not all the suppressions in the actual code are in this list. This list was also meant to be used for a rough prioritization of messages that need to be looked at, but with this change we are suppressing globally all C and many R, so the rest of the suppressions in the code do need to be looked at and addressed.

Reply:
>The one stop shop can instead be "pylint --list-msgs"

Agreed to this, always good to follow the standards of the tools we're using rather than keeping our own copy of it.

 >Not all the suppressions in the actual code are in this list.

Hmm interesting, I guess we missed some down the line. Anywho, I'm good with removing all these comments then (Y)

Reply:
OK, I just pushed another commit removing all the comments